test/suite/test_rollback_to_stable12.py

```diff
@@ -143,7 +143,7 @@ def test_rollback_to_stable(self):
         self.assertGreater(pages_visited, 0)
         self.assertGreaterEqual(hs_removed, 0)
         self.assertEqual(hs_sweep, 0)
-        self.assertGreater(pages_walk_skipped, 0)
+        self.assertGreaterEqual(pages_walk_skipped, 0)
```

Comment:
Why is this.

Reply:
The stats `txn_rts_tree_walk_skip_pages`can be 0 because of additional transaction id with existing stable timestamp check te rollback any update.